### PR TITLE
edited readme.md

### PR DESCRIPTION
**Pull request by evansusu**

### CODE REVIEW

**Comment by evansusu** [comment]
Help guys! I have no idea what is happening!

File: README.md [hunk 1]
```diff
@@ -13,12 +13,13 @@
 Browsers
 </th><td width=100%>
 
-Load `@octokit/plugin-rest-endpoint-methods` and [`@octokit/core`](https://github.com/octokit/core.js) (or core-compatible module) directly from [cdn.pika.dev](https://cdn.pika.dev)
+Load `@octokit/plugin-rest-endpoint-methods` and [`@octokit/core`](https://github.com/octokit/core.js) (or core-compatible module) directly from [cdn.skypack.dev](https://cdn.skypack.dev)
+
```
Comment (gr2m) [inline README.md:17]: you will have to remove the empty line. The tests are failing because it checks for file formatting using https://prettier.io/, too

**Review by gr2m — CHANGES_REQUESTED** [review]
You nearly got it. Sorry that our code formatter CI is so particular, it helps to keep our code base consistent despite having many contirbutors

File: README.md (hunk 1 above)
Comment (gr2m) [inline README.md:17]: Let me fix it for you 👍🏼 

```suggestion
Load `@octokit/plugin-rest-endpoint-methods` and [`@octokit/core`](https://github.com/octokit/core.js) (or core-compatible module) directly from [cdn.skypack.dev](https://cdn.skypack.dev)
```

Comment (gr2m) [reply README.md:17]: oh you did not give me access to your fork. Can you apply the suggestion above? You should see a big green button below the "Suggested change"

Comment (gr2m) [reply README.md]: nevermind it worked after all :)

**Review by gr2m — APPROVED** [review]
Changes look good now, yay! We will merge your changes shortly, together with other pull requests that address the same issue. We also added the `hacktoberfest-accepted` label so your pull request counts already, in case you are participating.

**Comment by gr2m** [comment]
Congratulations 🥳 We hope you enjoyed the process of sending the pull requests, and maybe learned a thing or two? If anything was unclear, or if you have any suggestions on how we can improve the suggestions in the issue #123, please let us know! It will help future first time contributors!

If you are looking to make another pull requests, look out for issues with the help wanted label. They are not as straight forward as the first-timers-only issues, but hey, you no longer are a first-timers 🙂 You can find help wanted issues [here](https://github.com/issues?q=is%3Aopen+is%3Aissue+org%3Aoctokit+label%3A%22help+wanted%22).
    
You can help us by starring this repository, as well as our primary repository at <https://github.com/octokit/rest.js/> ⭐️ Stars help others to discover our project.
    
If you are interested in Octokit and future opportunities to contribute, you can subscribe to our announcement discussion at <https://github.com/octokit/rest.js/discussions/620>. We will post and update soon and will point out issues you can contribute to.

**Pull request closed without merging**